src/i18n/locales/en.json

```diff
@@ -83,8 +83,9 @@
 		"home": "Home",
 		"docs": "Documentation",
 		"news": "News",
+		"download": "Download",
 		"themes": "Themes",
 		"discord": "Discord",
 		"github": "Github"
```

Comment:
Are those two still needed?

Reply:
I think they are used in the footer but didn't confirm

Reply:
No, the footer uses separate translation strings.